File: Advanced Web Technology/Lab/Lab-10 24-09 Assignment-10/Task1.php


<?php
class Vehicle {
    public $companyName;
    public $vehicleName;
    public $price;

    function set($companyName, $vehicleName, $price) {
        $this->companyName = $companyName;
        $this->vehicleName = $vehicleName;
        $this->price = $price;
    }

    public function displayDetails() {
        echo "Company Name: " . $this->companyName . "\n";
        echo "Vehicle Name: " . $this->vehicleName . "\n";
        echo "Price: $" . $this->price . "\n";
    }
}

$vehicle = new Vehicle();
$vehicle->set("Lego", "F1", 2400);
$vehicle->displayDetails();
?>

File: 2025-26/SEM4/Object Oriented Programming using C++/Lab/Solutions/Lab-6 18-02-26 Assignment 6/task1.cpp
// 1) Create a class Vehicle with the data members as Vehicle number, Vehicle Name, Vehicle Company name and Price.Define member functions to Accept and Display vehicle details.

#include <iostream>
#include <string>
using namespace std;

class Vehicle
{
public:

    string vno;
    string vname;
    string vcompany;
    float vprice;

    void accept()
    {
        cout << "Enter Vehicle Number: ";
        cin >> vno;
        cout << "Enter Vehicle Name: ";
        cin >> vname;
        cout << "Enter Vehicle Company Name: ";
        cin >> vcompany;
        cout << "Enter Vehicle Price: ";
        cin >> vprice;
    }

    void display()
    {
        cout << "Vehicle Number: " << vno << endl;
        cout << "Vehicle Name: " << vname << endl;
        cout << "Vehicle Company Name: " << vcompany << endl;
        cout << "Vehicle Price: " << vprice << endl;
    }
};

int main()
{
    Vehicle v;
    v.accept();
    v.display();
    return 0;
}